.github/workflows/create-cluster.yml

```diff
@@ -0,0 +1,355 @@
+name: Clusters
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: Version of the images
+        required: true
+        type: string
+      create-k8s-cluster:
+        description: Create a GKE demo cluster
+        type: boolean
+      create-os4-cluster:
+        description: Create an Openshift 4 demo cluster
+        type: boolean
+      create-long-cluster:
+        description: Create a long-running cluster on RC1
+        type: boolean
+      dry-run:
+        description: Dry-run
+        default: false
+        type: boolean
+
+env:
+  main_branch: ${{github.event.repository.default_branch}}
+  script_url: /repos/${{github.repository}}/contents/.github/workflows/scripts/common.sh?ref=${{ github.ref_name }}
+  DRY_RUN: ${{ fromJSON('["true", "false"]')[github.event.inputs.dry-run != 'true'] }}
+  ACCEPT_RAW: "Accept: application/vnd.github.v3.raw"
+  GH_TOKEN: ${{ github.token }}
+  GH_NO_UPDATE_NOTIFIER: 1
+  TIMEOUT_WAIT_FOR_IMAGES_SECONDS: 3600
+
+jobs:
+  properties:
+    runs-on: ubuntu-latest
+    outputs:
+      slack-channel: ${{ fromJSON(format('["{0}","{1}"]', steps.fetch.outputs.dry-slack-channel, steps.fetch.outputs.slack-channel))[github.event.inputs.dry-run != 'true'] }}
+      jira-projects: ${{ steps.fetch.outputs.jira-projects }}
+    steps:
+      - name: Read workflow properties file
+        id: fetch
+        env:
+          PROPERTIES_URL: /repos/${{ github.repository }}/contents/.github/properties?ref=${{ github.ref_name }}
+        run: gh api -H "$ACCEPT_RAW" "$PROPERTIES_URL" >> "$GITHUB_OUTPUT"
+
+  wait-for-images:
+    name: Wait for images on Quay.io
+    runs-on: ubuntu-latest
+    if: >- # Skip if no clusters are going to be created.
+      inputs.create-k8s-cluster != 'false' ||
+      inputs.create-os4-cluster != 'false' ||
+      inputs.create-long-cluster != 'false'
+    strategy:
+      matrix:
+        image: [main, scanner, scanner-db, collector]
+    steps:
+      - name: Wait for the ${{matrix.image}} image
+        uses: stackrox/actions/release/wait-for-image@v1
```

Comment:
I am not 100% convinced with hardcoding the reference for the action.
Here is what did not work:
- checking out the repository and using a local action, which would need to pass the workflow ref from the calling workflow to here. For unknown reasons the action behaves differently when called locally or as shell script vs as a remote action, and the `gh_log` output becomes part of the block output [here](https://github.com/stackrox/actions/blob/main/release/wait-for-image/wait-for-image.sh#L33), which causes jq and the loop to fail.
- using an input (or other) variable in the path, e.g. `stackrox/actions/release/wait-for-image@${{ inputs.workflow-ref }}`

The other actions in this workflow are not affected by the output bug. 
If we remove the `gh_log` output from the block `wait-for-image`, we could use the checkout and local action approach (with passing the workflow ref from the calling workflow).

Reply:
> The other actions in this workflow are not affected by the output bug.
If we remove the gh_log output from the block wait-for-image, we could use the checkout and local action approach (with passing the workflow ref from the calling workflow).

fixed in #18

Reply:
@vikin91 and I agreed offline to pass the `workflow-ref` to checkout the `actions` repo and run a local action.